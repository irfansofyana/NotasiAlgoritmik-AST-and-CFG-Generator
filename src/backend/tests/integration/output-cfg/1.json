{
   "nodes":[
      {
         "label":1,
         "statements":[
            "input(A,B)"
         ]
      },
      {
         "label":2,
         "statements": [
            "output(A + B)"
         ]
      },
      {
         "label": 3,
         "statements": [
            "start_algorithm"
         ]
      },
      {
         "label": 4,
         "statements": [
            "end_algorithm"
         ]
      }
   ],
   "edges":[
      {
         "start_node_label":1,
         "end_node_label":2
      },
      {
         "start_node_label": 2,
         "end_node_label": 4
      },
      {
         "start_node_label": 3,
         "end_node_label": 1
      }
   ]
}