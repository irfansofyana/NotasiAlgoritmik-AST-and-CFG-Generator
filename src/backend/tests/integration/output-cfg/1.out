// CFG result
digraph {
	1 [label="input(A,B)"]
	1 -> 2
	2 [label="output(A + B)"]
	2 -> 5
	4 [label=start_algorithm]
	4 -> 1
	5 [label=end_algorithm]
}